src/components/books/Book.module.css

```diff
@@ -1,3 +1,35 @@
 .book {
-  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+  background-color: white;
+  width: 85%;
+  padding: 20px;
+  padding-inline: 40px;
+}
+
+.title_author {
+  width: 40%;
+  border-right: 1px solid rgb(68, 67, 67);
+}
+
+.title_author h3 {
+  font-size: 25px;
+}
+
+.title_author p {
+  font-size: 17px;
+  color: #037cd8;
+}
+
+button {
+  border: none;
+  padding-inline: 20px;
+  padding-block: 10px;
+  background-color: #0290ff;
+  color: white;
+  font-size: 15px;
+  border-radius: 3px;
+  cursor: pointer;
 }
```

Comment:
### Your Project Design 🔴
![Capture](https://user-images.githubusercontent.com/93039359/208040586-17175130-eeef-429d-8b11-80d0d301f6d7.PNG)

### Required Project Design 🟢
![bookstore - Copy](https://user-images.githubusercontent.com/93039359/208041544-bdbe990e-7b68-4bd8-8cf8-b6f29d439242.png)

- [ ] You can compare both designs (Yours and Template) and you will find that a lot of things are missing in your design such as
- Book Genre (Shown by Yellow 🟡 Highlight)
- Progress Circle (Shown by Red 🔴 Highlight)
- Comment, Remove, Edit Buttons (Shown by Red 🔴 Highlight)
- The remove button which you have added is at the wrong place 😨 That place belongs to update progress buttons and its details 🤔(Shown by Red 🔴 Highlight)